build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
     testImplementation(kotlin("test"))
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
     testImplementation("io.mockk:mockk:1.12.4")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
```

Comment:
Why did u add the junit dependancy again? it already exists.
```suggestion
```

Reply:
done